recipes/qgis/patches/auth.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/core/auth/qgsauthmethodregistry.cpp b/src/core/auth/qgsauthmethodregistry.cpp
+index a5df195189..429620ab7d 100644
+--- a/src/core/auth/qgsauthmethodregistry.cpp
++++ b/src/core/auth/qgsauthmethodregistry.cpp
+@@ -60,7 +60,7 @@ QgsAuthMethodRegistry::QgsAuthMethodRegistry( const QString &pluginPath )
+ #if defined(Q_OS_WIN) || defined(__CYGWIN__)
+   mLibraryDirectory.setNameFilters( QStringList( "*authmethod.dll" ) );
```

Comment:
Also here?

Reply:
Do we? That's windows specific, I'm not in the know ;)